test/regfox/regfox_api.test.js

```diff
@@ -136,4 +154,36 @@ describe('regfox_api', () => {
       return login('anything', 'none').should.eventually.be.rejected;
     });
   });
+
+  describe('getRegistrationInfo', () => {
+    const getResponse = stub();
+    const server = setupServer(...getRegistrationInfoHandler(123455, getResponse));
+    before(() => server.listen());
+    afterEach(() => getResponse.reset());
+    afterEach(() => server.resetHandlers());
+    after(() => server.close());
+
+    it('returns data as requested', async () => {
+      getResponse.returns({
+        "code": 200,
+        "error": null,
+        "data": {
+          "id": 123455,
+          "accountId": 1311,
+          "registrationId": "593409403",
+          "transactionCount": 1
+        }
+      });
+      const result = await getRegistrationInfo(123455);
+
+      expect(result).to.include({ 'transactionCount': 1 });
```

Comment:
Moot to check for other details?

Valuable to do any type checking (ID is a number vs string)?

Reply:
I'm happy to check for other properties being present in the response (admittedly I was fairly lax when checking here). Was there some particular ones you wanted to include?

But I don't think we should be /too/ tightly coupled that if they change a type (but semantically an id is still an id), our code fails. (Now, canary tests would be a different story, but I don't think we can set those up). Basically since javascript is pretty type lenient, I'm following those same practices.